lang/swi-prolog-devel/Portfile

```diff
@@ -1,14 +1,16 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
```

Comment:
The Portfile does not conform to this modeline (but it should). Please make a second commit in which you modify the whitespace to conform to this modeline. Ideally, make sure that there are as few differences as possible between swi-prolog and swi-prolog-devel. For example, lines should be in the same order, columns should be indented at the same number of spaces, etc.

Reply:
This is getting a little tedious.  I don't have any experience with this tools and guidelines.  I was asked to add this mode line.  I use neither of these editors.   The two Portfiles are now as much as possible in sync.   I'd prefer if you make it once the way you want it and I'll make sure future changes are minimal.

Reply:
Since you are adding `openmaintainer`, one of us can go in and do that now.

Reply:
I'll try to remember to do that after this is committed.

Reply:
It will be much appreciated if you take care of the little details one time.  Once that is done I assume the process will be smooth.  Getting the first time right seems hard though and in my experience with PRs, it is way quicker as maintainer to finish the last little bits than trying to explain a novice of all tiny details.